extras/printschema.py

```diff
@@ -0,0 +1,45 @@
+import copy
```

Comment:
wrt Git: This file is small, so it doesn't bother me that it'll stay in git history. If you want to remove it from history though, you can use a rebase like [this](http://sethrobertson.github.io/GitFixUm/fixup.html#remove_deep). It can get kinda messy though, especially since you also modified the `.gitignore`. Plus you'd need to force-push, which would mess up this diff. IMHO, all this trouble isn't worth it.